### DISCUSSION TITLE
Usage Questions

### DISCUSSION THREAD

**Discussion by mrobe** [ScanTailor-Advanced/scantailor-advanced#56] [Q&A]
Trying to learn how to use STA, I'm stumbling over a series of things. Perhaps there are procedures I'm not getting yet(?).

**(1)** I would like to set the output DPI to be the same as the input DPI. The way STA works now, it seems that I have to check the DPI of my source images and dial in the same number as a 'Custom' setting, and then apply it to every page.

STA doesn't seem report the DPI of the source images, so I have to use a different app for this.

I tried fiddling around with 'Fix DPI...' but this caused strange results.

Question: is there a better way to do this?

Ideally, there would be a setting to just use the same DPI as the source files.

**(2)** After I have all my parameters set up, I click the 'Run' button for 'Output'. STA generates output, but only for one page. It seems like I have to manually click this for each page of output.

Question: is there a better way to do this? How should I generate output for all pages?

**(3)** Among the 'Output' parameters, I find a 'Mixed' mode.

Question: what is this and how should I use it?

E.g., Some pages in my scan have a combination of text and images. These often seem to need different processing, as the settings to get crisp text are different from those to get clear images. Is that what this setting is for?

**(4)** I would like the text in the output scans to be centered on each page. I have tried 'Margins' > 'Alignment' > 'Horizontal mode' > 'Auto', but it doesn't work. On some pages, the text is still much closer to one side of the page.

Question: how can I make this work?

**(5)** My input images are RGB, but for output I'd like monochrome, or even a bitmap with diffusion dither, to reduce file size.

Question: is there a way to do this with STA?

Thanks!

**Comment by zvezdochiot** [comment]
@mrobe say:
>(5) My input images are RGB, but for output I'd like monochrome, or even a bitmap with diffusion dither, to reduce file size.
>Question: is there a way to do this with STA?

See:
* https://github.com/ScanTailor-Advanced/scantailor-advanced/pull/48 .
* https://github.com/ImageProcessing-ElectronicPublications/scantailor-advanced/releases/tag/v1.0.18

**Reply by mrobe** [reply]
I don't understand this, but thank you :'D

**Comment by mrobe** [comment]
Bump.
